src/test/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidatorUTest.java

```diff
@@ -117,11 +117,11 @@ public void validate_rejectBadAssignmentPoints() {
     public void validate_rejectNullPoints() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)